libs/dh/market-participant/feature-gridarea/src/lib/dh-market-participant-gridarea.component.ts

```diff
@@ -26,53 +26,29 @@ import {
   WattValidationMessageModule,
 } from '@energinet-datahub/watt';
 import { Router } from '@angular/router';
-import {
-  dhMarketParticipantGridAreasCreatePath,
-  dhMarketParticipantGridAreasEditPath,
-  dhMarketParticipantGridAreasPath,
-  dhMarketParticipantPath,
-} from '@energinet-datahub/dh/market-participant/routing';
 import { DhMarketParticipantGridAreaOverviewScam } from './overview/dh-market-participant-gridarea-overview.component';
 import { PushModule } from '@rx-angular/template';
 
 @Component({
   selector: 'dh-market-participant-gridarea',
   styleUrls: ['./dh-market-participant-gridarea.component.scss'],
   templateUrl: './dh-market-participant-gridarea.component.html',
-  providers: [DhMarketParticipantOverviewDataAccessApiStore],
+  providers: [DhMarketParticipantGridAreaOverviewDataAccessApiStore],
 })
 export class DhMarketParticipantGridAreaComponent {
   constructor(
-    private store: DhMarketParticipantOverviewDataAccessApiStore,
+    private store: DhMarketParticipantGridAreaOverviewDataAccessApiStore,
     private router: Router
```

Comment:
issue: the `router` is not used. Can be removed. Will give a lint error otherwise.